src/math/Box3.js

```diff
@@ -435,20 +435,42 @@ class Box3 {
 
 	}
 
+	getVertexPoints( points ) {
+
+		points.length = 0;
+
+		points.push( new Vector3(), new Vector3(), new Vector3(), new Vector3(), new Vector3(), new Vector3(), new Vector3(), new Vector3() );
```

Comment:
I'm afraid we can't implement it like this since `applyMatrix4()` is often called per frame multiple times and creating instances of `Vector3` over and over again will introduce performance issues. Especially when the GC needs to clean things up.